packages/bun-usockets/src/crypto/openssl.c

```diff
@@ -647,6 +949,7 @@ X509_STORE* us_get_default_ca_store() {
     }
     
     us_internal_init_root_certs();
+    us_internal_init_native_certs();
```

Comment:
If we do this, we should just use the native certs and not both right? So on macOS, it would only load native certs and on Linux, it would only load root certs. 

I think that we also should see if there's any perf difference in load time as I remember a 30ms perf hit a long time ago when loading native certs using a non-OpenSSL/BoringSSL library and loading from the system trust store.

Reply:
I just did some profiling, and the results are not good. The 3 `SecTrustSettingsCopyCertificates` requests to get all the certificates took 70ms total & the `SecTrustSettingsCopyTrustSettings` on each certificate to determine whether the system trusts the cert took 78ms total. As of 1.0.6, bun only takes 6ms to load certificates.

That's really disappointing. I was not expecting these APIs to be so slow.

I think a better approach would be to follow what Deno did in denoland/deno#11491. On all platforms, default to embedded Mozilla certs, and have some flag/env variable/config to enable the use of native certs for those who want/need it and are willing to take the first request performance hit.